spacy/tests/serialize/test_serialize_kb.py

```diff
@@ -0,0 +1,73 @@
+# coding: utf-8
+from ..util import make_tempdir
+from ...util import ensure_path
+
+from spacy.kb import KnowledgeBase
+
+
+def test_serialize_kb_disk(en_vocab):
+    # baseline assertions
+    kb1 = _get_dummy_kb(en_vocab)
+    _check_kb(kb1)
+
+    # dumping to file & loading back in
+    with make_tempdir() as d:
+        dir_path = ensure_path(d)
+        if not dir_path.exists():
+            dir_path.mkdir()
+        file_path = dir_path / "kb"
+        print(file_path, type(file_path))
+        kb1.dump(str(file_path))
+
+        kb2 = KnowledgeBase(vocab=en_vocab, entity_vector_length=3)
+        kb2.load_bulk(str(file_path))
+
+    # final assertions
+    _check_kb(kb2)
+
+
+def _get_dummy_kb(vocab):
+    kb = KnowledgeBase(vocab=vocab, entity_vector_length=3)
+
+    kb.add_entity(entity=u'Q53', prob=0.33, entity_vector=[0, 5, 3])
+    kb.add_entity(entity=u'Q17', prob=0.2, entity_vector=[7, 1, 0])
+    kb.add_entity(entity=u'Q007', prob=0.7, entity_vector=[0, 0, 7])
+    kb.add_entity(entity=u'Q44', prob=0.4, entity_vector=[4, 4, 4])
+
+    kb.add_alias(alias=u'double07', entities=[u'Q17', u'Q007'], probabilities=[0.1, 0.9])
+    kb.add_alias(alias=u'guy', entities=[u'Q53', u'Q007', u'Q17', u'Q44'], probabilities=[0.3, 0.3, 0.2, 0.1])
+    kb.add_alias(alias=u'random', entities=[u'Q007'], probabilities=[1.0])
+
+    return kb
```

Comment:
Great test here! I often struggle to build these small test cases.

Reply:
We can totally leave out the `u` strings in our own code btw if the files all have the correct declarations:

```python
# coding: utf-8
from __future__ import unicode_literals 
```